upgrading/automated_upgrades.adoc

```diff
@@ -328,8 +328,26 @@ $ ansible-playbook -i <path/to/inventory/file> \
     -e openshift_upgrade_nodes_label="region=group1"
 ----
 
-See xref:../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Managing Nodes] for more on node labels.
+The `openshift_upgrade_nodes_max_fail_percentage` variable allows you to specify
+how many nodes may fail in each batch. The percentage of failure must exceed
+your value before the playbook aborts the upgrade.
 
+The `openshift_upgrade_nodes_drain_timeout` variable allows you to specify how
+long to wait for the node to drain before forcefully killing the node.
+
+In this example, 10 nodes are upgraded at a time, the upgrade will abort if 20
```

Comment:
Lets change this to "if more than 20 percent" as if you hit 20% exactly it won't abort. So in this scenario it will only abort as soon as the third node fails to upgrade properly.